.ci/ci-fast-return.sh

```diff
@@ -27,7 +27,7 @@ source "${cidir}/lib.sh"
 
 # If no branch specified, compare against the main branch.
 # The 'branch' var is required by the get_pr() lib functions.
-branch=${branch:-main}
+branch=${branch:-stable-2.4}
```

Comment:
Is this something we will have to do on every branch bump?

Reply:
I don't have an answer right now, I will have to look at the scripts and see if there is a way to read branch value from the same source.

Just to be clear, if we drop those changes than the QEMU problem is still fixed by the other commit (https://github.com/kata-containers/tests/pull/4807/commits/fc20fff6bb4d9489668420cabbf41b64c1b254ca).

Reply:
done.